src/common/io_util.cc

```diff
@@ -153,8 +153,7 @@ Status SockConnect(const std::string &host, uint32_t port, int *fd, int conn_tim
         return Status(Status::NotOK, std::string("setsockopt failed: ") + strerror(errno));
```

Comment:
Could you use `Status::FromErrno` here as well?

Reply:
There are so many code that can be replaced to these new methods, so I think it is not a main point of this PR.